cirq-core/cirq/qis/states_test.py

```diff
@@ -475,6 +475,13 @@ def test_dirac_notation_precision():
     assert_dirac_notation_python([sqrt, sqrt], "0.707|0⟩ + 0.707|1⟩", decimals=3)
 
 
+def test_dirac_notation_invalid():
+    with pytest.raises(ValueError):
```

Comment:
I would probably add a `match="'state_vector has incorrect size"` to the pytest.raises so we make sure and catch the right ValueError.